app/assets/javascripts/admin/addon/controllers/modals/admin-install-theme.js

```diff
@@ -1,46 +1,51 @@
 import { COMPONENTS, THEMES } from "admin/models/theme";
 import Controller, { inject as controller } from "@ember/controller";
-import { alias, equal, match } from "@ember/object/computed";
-import discourseComputed, { observes } from "discourse-common/utils/decorators";
+import discourseComputed from "discourse-common/utils/decorators";
+import { observes } from "@ember-decorators/object";
 import I18n from "I18n";
 import ModalFunctionality from "discourse/mixins/modal-functionality";
 import { POPULAR_THEMES } from "discourse-common/helpers/popular-themes";
 import { ajax } from "discourse/lib/ajax";
 import { popupAjaxError } from "discourse/lib/ajax-error";
-import { set } from "@ember/object";
+import { action, set } from "@ember/object";
 
 const MIN_NAME_LENGTH = 4;
 
-export default Controller.extend(ModalFunctionality, {
-  adminCustomizeThemes: controller(),
-  themesController: controller("adminCustomizeThemes"),
-  popular: equal("selection", "popular"),
-  local: equal("selection", "local"),
-  remote: equal("selection", "remote"),
-  create: equal("selection", "create"),
-  directRepoInstall: equal("selection", "directRepoInstall"),
-  selection: "popular",
-  loading: false,
-  keyGenUrl: "/admin/themes/generate_key_pair",
-  importUrl: "/admin/themes/import",
-  recordType: "theme",
-  checkPrivate: match("uploadUrl", /^ssh:\/\/.+@.+$|.+@.+:.+$/),
-  localFile: null,
-  uploadUrl: null,
-  uploadName: null,
-  advancedVisible: false,
-  selectedType: alias("themesController.currentTab"),
-  component: equal("selectedType", COMPONENTS),
-  urlPlaceholder: "https://github.com/discourse/sample_theme",
-
-  init() {
-    this._super(...arguments);
-
-    this.createTypes = [
-      { name: I18n.t("admin.customize.theme.theme"), value: THEMES },
-      { name: I18n.t("admin.customize.theme.component"), value: COMPONENTS },
-    ];
-  },
+export default class AdminInstallThemeController extends Controller.extend(
+  ModalFunctionality
+) {
+  @controller adminCustomizeThemes;
+  @controller("adminCustomizeThemes") themesController;
+
+  @equal("selection", "popular") popular;
+  @equal("selection", "local") local;
+  @equal("selection", "remote") remote;
+  @equal("selection", "create") create;
+  @equal("selection", "directRepoInstall") directRepoInstall;
+
+  selection = "popular";
+  loading = false;
+  keyGenUrl = "/admin/themes/generate_key_pair";
+  importUrl = "/admin/themes/import";
+  recordType = "theme";
+
+  @match("uploadUrl", /^ssh:\/\/.+@.+$|.+@.+:.+$/) checkPrivate;
+
+  localFile = null;
+  uploadUrl = null;
+  uploadName = null;
+  advancedVisible = false;
+
+  @alias("themesController.currentTab") selectedType;
+  @equal("selectedType", COMPONENTS) component;
+
+  urlPlaceholder = "https://github.com/discourse/sample_theme";
+
+  createTypes = [
+    { name: I18n.t("admin.customize.theme.theme"), value: THEMES },
+    { name: I18n.t("admin.customize.theme.component"), value: COMPONENTS },
+  ];
```

Comment:
Did the codemod split these into groups? 😃 (update: or probably prettier/eslint, after the codemod conversion)

Reply:
Yeah the codemod/eslint added linebreaks between **all** of them. I removed some of those linebreaks which gave us this grouping. Have now pushed a commit to remove the grouping so it's closer to the original layout.